release-tools/build.make

```diff
@@ -60,9 +60,27 @@ endif
 # Specific packages can be excluded from each of the tests below by setting the *_FILTER_CMD variables
 # to something like "| grep -v 'github.com/kubernetes-csi/project/pkg/foobar'". See usage below.
 
-build-%:
+BUILD_IN_DOCKER?=false
+DOCKER_BUILDER_WD=/go/src/github.com/kubernetes-csi/external-attacher
```

Comment:
This file is shared among repos via csi-release-tools (see README.md). One cannot hard-code individual repo names here.

Reply:
OK, lemme see what I can do about that...